app/server/load-data.js

```diff
@@ -11,15 +13,32 @@ import { catalogDataLoader } from "@quintype/framework/server/data-loader-helper
 import { getNavigationMenuArray } from "./data-loaders/menu-data";
 import { PAGE_TYPE } from "../isomorphic/constants";
 
-const WHITELIST_CONFIG_KEYS = ["cdn-image", "polltype-host", "layout", "sections", "social-links", "publisher-name"];
+const WHITELIST_CONFIG_KEYS = [
+  "cdn-image",
+  "polltype-host",
+  "layout",
+  "sections",
+  "social-links",
+  "publisher-name",
+  "sketches-host",
+  "publisher-settings"
+];
+
+export function getPublisherAttributes(config, publisherYml = publisher) {
```

Comment:
remove config, it's not being used in this function

Reply:
done